server/bin/gold/test-5.2.txt

```diff
@@ -316,6 +316,17 @@ len(actions) = 1
 --- Finished pbench-verify-backup-tarballs (status=0)
 +++ Running pbench-satellite-cleanup
 --- Finished pbench-satellite-cleanup (status=0)
++++ Running pbench-tarball-stats
+Took 0.00 seconds
+                 0 controllers with bad tar balls
+                 0 bad tar balls, total count
+                 0 bad tar balls, total size
+                 6 controllers with good tar balls (-INT- 3, ONE 3)
```

Comment:
Wow, that `-INT-` and `ONE` make **no** sense at all until reading the code. (The fact that the test cases are using satellite `ONE` where we usually only see `EC2` probably doesn't help.)

I don't know if some keyword introducing the breakdown, like maybe "origin", would help. I suppose in a real installation, the satellite prefix would be commonly recognized and that might help explain the `-INT-` but something like "local" might be better. 😬

Reply:
> Wow, that `-INT-` and `ONE` make **no** sense at all until reading the code. (The fact that the test cases are using satellite `ONE` where we usually only see `EC2` probably doesn't help.)

Well, it is better to not hard-code a particular satellite name.

> I don't know if some keyword introducing the breakdown, like maybe "origin", would help. I suppose in a real installation, the satellite prefix would be commonly recognized and that might help explain the `-INT-` but something like "local" might be better. 😬

Naming ...

Reply:
What if the output looked like:

```
Took 0.00 seconds
Summary Statistics
                 0 controllers with bad tar balls
                 0 bad tar balls (total count)
                 0 bad tar balls (total size)
                 6 controllers with good tar balls
                 3 main pbench server
                 3 "ONE" pbench satellite server
                 8 good tar balls (total count)
                 5 main pbench server
                 3 "ONE" pbench satellite server
             1,764 good tar balls, total size

Tar Ball Counts broken down by Year and Month
     Total Count  % main   % ONE
1970:          8  62.50%  37.50%
1970-01:       8  62.50%  37.50%
```

Reply:
Or even:
```
Took 0.00 seconds
Summary Statistics
    Good Tar Balls
                 8 good tar balls total count
                 5 main pbench server
                 3 "ONE" pbench satellite server
             1,764 good tar balls total size

    Good Tar Balls - controller counts
                 6 controllers with good tar balls
                 3 main pbench server
                 3 "ONE" pbench satellite server

    Bad Tar Balls
                 0 controllers with bad tar balls
                 0 bad tar balls total count
                 0 bad tar balls total size

Tar Ball Counts broken down by Year and Month, with satellite percentages
Year/Month  Total Count  % main   % ONE
1970                  8  62.50%  37.50%
1970-01               8  62.50%  37.50%
```

Reply:
I second the request for better names and/or more context.  I like the first of the two proposals, although I would ask for the subtotals to be indented under the totals.  The second proposal is good, too, but it feels kind of big and "airy" -- I appreciate the compactness of the earlier ones (although, I do like the aligned columns for the temporal breakdowns).

Reply:
I kind like the big and airy ...

```
Summary Statistics for Tar Balls (external data only):

    Took 0.00 seconds to find all tar balls.

    Good Tar Balls:
                 8 total count
             1,764 total size

        By Server Origin:
                 5 main pbench server
                 3 "ONE" pbench satellite server

        Controller Counts:
                 6 controllers
                 3 main pbench server
                 3 "ONE" pbench satellite server

    Bad Tar Balls:
                 0 controllers
                 0 total count
                 0 total size


Tar Ball Counts broken down by Year and Month, with satellite percentages:

Year/Month  Total Count  % main   % ONE
1970                  8  62.50%  37.50%
1970-01               8  62.50%  37.50%
```

Reply:
> I kind like the big and airy ...

Good -- let's go with that, then.